components/mixins/profile/profile-consents.example.1.json

```diff
@@ -54,6 +79,16 @@
         "john@xyz.com": {
           "xdm:marketing": {
             "xdm:email": {
```

Comment:
Previously, at this place in the idSpecific section, I could have specified the "xdm:preferred" and "xdm:any" fields. With this update they are no longer allowed here, but as this example file shows, they are still supported above at lines 15 and 16.

Reply:
Actually this comment should have been on line 80 instead of 81